lib/uklock/include/uk/mutex.h

```diff
@@ -57,6 +60,31 @@ struct uk_mutex {
 	struct uk_waitq wait;
 };
 
+/*
+ * Mutex statistics for ukstore.
+ */
+struct uk_mutex_metrics {
+	/** Mutex objects currently locked */
+	size_t active_locked;
+	/** Mutex objects currently unlocked */
+	size_t active_unlocked;
+
+	/** Successful blocking lock operations since startup */
+	size_t total_locks;
+	/** Successful non-blocking (trylock) operations since startup */
+	size_t total_ok_trylocks;
+	/** Failed non-blocking (trylock) operations since startup */
+	size_t total_failed_trylocks;
+	/** Successful unlock operations since startup */
+	size_t total_unlocks;
+};
+
+/*
+ * Metric storage (see mutex.c).
+ */
+extern struct uk_mutex_metrics mutex_metrics;
+extern spinlock_t              mutex_metrics_lock;
```

Comment:
I would prefix these like (and their other appearances, of course):
```
mutex_metrics;        -> _uk_mutex_metrics;
mutex_metrics_lock;   -> _uk_mutex_metrics_lock;
```

This is to ensure that there won't be name conflicts in the future (someone might create another `mutex_metrics` symbol).